pkg/sql/sem/builtins/pg_builtins.go

```diff
@@ -1942,6 +1942,8 @@ func getCatalogOidForComments(catalogName string) (id int, ok bool) {
 		return catconstants.PgCatalogClassTableID, true
 	case "pg_database":
 		return catconstants.PgCatalogDatabaseTableID, true
+	case "pg_schema":
+		return catconstants.PgCatalogSchemaID, true
```

Comment:
I checked the postgres documentation and I couldn't find any mention of `pg_schema` in there. Strictly speaking the table for schemas in the pg catalog is `pg_namespace`. As far as comments are concerned, it seems they might actually belong in `pg_shdescription` or `pg_description`. Perhaps @rafiss can enlighten us?

Reply:
a quick test inside `psql` tells me that schema comments get stored inside `pg_description`.